pinc/forum_interface_phpbb3.inc

```diff
@@ -776,47 +778,29 @@ function call_phpbb_function($func_name, $args)
         return;
     }
 
-    $escaped_args_str = implode(' ', array_map('my_escapeshellarg', $args));
-
     // (cd into the pinc directory, so the script doesn't
     // have to guess where it's being invoked from.)
     global $code_dir, $php_cli_executable;
```

Comment:
This is the only file that uses `$php_cli_executable;` and Symfony has [PhpSubprocess](https://symfony.com/doc/current/components/process.html#executing-a-php-child-process-with-the-same-configuration) that automatically finds the PHP executable. It would be nice to see if that works for us and if so remove `$php_cli_executable` from here, `site_vars.php.template`, and the whole section in `configuration.sh` that tries to find the executable.

Reply:
Good idea, I'll try it.

Reply:
We are using symfony/process version 5.4, which does not include the PhpSubprocess class. It looks like this was introduced in version 6. But version 6 requires php 8. My dev VM has php 7.4.3, is that the version I should have?

Reply:
Ah, I missed that. Something for us to consider down the road when we're on PHP 8. Your VM has the right version of PHP on it 👍